src/events.h

```diff
@@ -60,6 +59,13 @@ namespace events {
     };
     extern signal<void(StickData &)> stick_event;
 
+    // Audio
+    struct AudioData {
+        std::string filename;
+    };
```

Comment:
wait, he said this in the pr description: "For AudioData, I might add future variables for volume, looped, etc. so we can have more fine-grain control for both the music and sound effects."